src/eatery_db/campus_eatery.py

```diff
@@ -132,11 +131,114 @@ def parse_menu_items(items_json, category_model):
+        latitude, longitude = parse_coordinates(eatery)
+
+        new_eatery = CampusEatery(
+            about=eatery.get("about", ""),
+            campus_area_desc=parse_campus_area(eatery),
+            eatery_type=parse_eatery_type(eatery),
+            image_url=get_image_url(eatery.get("slug", "")),
+            latitude=latitude,
+            location=eatery.get("location", ""),
+            longitude=longitude,
+            name=eatery.get("name", ""),
+            name_short=eatery.get("nameshort", ""),
+            payment_method_brbs=brbs,
+            payment_method_cash=cash,
+            payment_method_cornell_card=cornell_card,
+            payment_method_credit=credit,
+            payment_method_mobile=mobile,
+            payment_method_swipes=swipes,
+            phone=eatery.get("contactPhone", "N/A"),
+            slug=eatery.get("slug", ""),
+        )
+        static_eateries.append(new_eatery)
+
+    return static_eateries
+
+
+def parse_static_op_hours(data_json, eatery_model):
+    """Parses a Cornell Dining json dictionary.
+
+    Returns a list of tuples of CampusEateryHour objects for a corresponding CampusEatery object and their unparsed
+    menu.
+
+    Args:
+        data_json (dict): a valid dictionary from the Cornell Dining json
+        eatery_model (CampusEatery): the CampusEatery object to which to link the hours.
+    """
+    today = date.today()
+    for eatery in data_json["eateries"]:
+        if eatery_model.slug == eatery.get("slug", ""):
+            weekdays = {}
+
+            hours_list = eatery.get("operatingHours", [])
+            dining_items = eatery.get("diningItems")
+            dates_closed = eatery.get("datesClosed", [])
+
+            for hours in hours_list:
+                if "-" in hours["weekday"]:
+                    start, end = hours["weekday"].split("-")
+                    start_index = WEEKDAYS[start]
+                    end_index = WEEKDAYS[end] + 1
+                    days = list(range(start_index, end_index))
+                else:
+                    days = [WEEKDAYS[hours["weekday"]]]
+                for weekday in days:
+                    if weekday not in weekdays:
+                        weekdays[weekday] = hours["events"]
+
+            new_operating_hours = []
+            for i in range(NUM_DAYS_STORED_IN_DB):
+                new_date = today + timedelta(days=i)
+                for dates in dates_closed:  # check if dates_closed contains new_date
+                    if "-" in dates:  # indicates this string is a date range of form "mm/dd/yy-mm/dd/yy"
+                        start_date, end_date = string_to_date_range(dates)
+                        if start_date <= new_date <= end_date:
+                            break
+                    else:  # date string is a singular date
+                        closed_date = datetime.strptime(dates, "%m/%d/%y").date()
```

Comment:
we have this snippet: `datetime.strptime(var_name, "%m/%d/%y")` very frequently, this should be moved into a helper function so that these calls are unified

Reply:
This specific format is only 3 times, is it still worth making a function for it?